src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/resource/NonAssociationsResource.kt

```diff
@@ -527,6 +531,37 @@ class NonAssociationsResource(
     }
   }
 
+  @GetMapping("/non-associations/paged")
```

Comment:
could this pls be simply at "/non-associations" so that there's a tidier structure?

Reply:
@ushkarev I've updated